examples/nodejs/readme.ts

```diff
@@ -0,0 +1,22 @@
+import {CacheGet, CacheClient, Configurations, CredentialProvider} from '@gomomento/sdk';
+
+async function main() {
+  const momento = new CacheClient({
+    configuration: Configurations.Laptop.v1(),
+    credentialProvider: CredentialProvider.fromEnvironmentVariable({
+      environmentVariableName: 'MOMENTO_AUTH_TOKEN',
+    }),
+    defaultTtlSeconds: 60,
+  });
+
+  const createCacheResponse = await momento.createCache('cache');
+  const setResponse = await momento.set('cache', 'foo', 'FOO');
```

Comment:
kinda agree w/ lint warnings maybe better to just not init a response if not checking it at all? 
`await momento.set('cache', 'foo', 'FOO');`

Reply:
dont feel super strongly on this one also ok w/ merging as is as well and can decide on another pr if want to remove or not.

Reply:
Should be a short page I can whip up. We have two styles of error handling in our SDKs, right? One for what returns from a data retrieval operations, hit/miss/error, and one is for other operations success and error.

Reply:
@NoSQLKnowHow what i was planning on doing is porting over the error handling section from the old readme as a starting point.  I also want to include some stuff about config and credentials that was in there.  I am planning to throw together something rough today as a starting point and then you can help me figure out where you think it should fit in to the site